src/test/data/JsonSerializableAddressBookTest/duplicatePersonAddressBook.json

```diff
@@ -3,25 +3,15 @@
     "name": "Alice Pauline",
     "phone": "94351253",
     "email": "alice@example.com",
-    "personEvents": [{
-      "name": "IFG"
-    }],
-    "roles": [ "friends" ]
+    "eventRoles": {
+      "IFG": ["friends"]
+    }
   }, {
     "name": "Alice Pauline",
     "phone": "94351253",
     "email": "pauline@example.com",
-    "personEvents": [{
-      "name": "IFG"
-    }]
-  } ],
-  "events": [ {
-    "name": "IFG"
-  }, {
-    "name": "AUG"
-  }, {
-    "name": "SUNIG"
-  }, {
-    "name": "IVP"
```

Comment:
We still need the "events" row to show the events separately on the right panel as we discussed, can we not delete this one?